README.md

```diff
@@ -31,8 +31,34 @@ This bundle is for multi-node deployments, for individual deployments for
 developers, use the smaller
 [kubernetes-core](http://jujucharms.com/kubernetes-core) bundle.
 
-## Deploy the bundle
+## Proxy configuration
+
+If you're operating behind a proxy, you'll need to configure your model
+appropriately before deploying the Kubernetes bundle.
 
+First, configure your model's `http-proxy` and `https-proxy` settings with your
+proxy (here we use `squid.internal:3128` as an example):
+
+```sh
+$ juju model-config http-proxy=http://squid.internal:3128 https-proxy=https://squid.internal:3128
+```
+
+You'll also need to add `localhost` to the `no-proxy` model configuration
```

Comment:
Explain what no-proxy does and why they need it.  "You will need to add some hosts that are not proxied, such as localhost, and the address(es) of the local Juju controller."